### ISSUE TITLE
quickValidate in service.validate.ts is generated wrong

### ISSUE THREAD

**Issue by murbanowicz** [feathers-plus/generator-feathers-plus#52]
Generated code:
```js
let quickValidate = (method: string, data: any, options?: any) => {
  try {
    if (method === 'create') { validateCreate(options)({ type: 'before', method: 'create', data }); }
    if (method === 'update') { validateCreate(options)({ type: 'before', method: 'update', data }); }
    if (method === 'patch') { validateCreate(options)({ type: 'before', method: 'patch', data }); }
  } catch (err) {
    return err;
  }
};
```

**Error:**
`TS2345: Argument of type '{ type: "before"; method: string; data: any; }' is not assignable to parameter of type 'HookContext<any>'.   Property 'app' is missing in type '{ type: "before"; method: string; data: any; }'.`

**Comment by eddyystop** [comment]
@j2L4e  ^^^

**Comment by j2L4e** [comment]
Dupe of #49, will be back tomorrow night

**Comment by eddyystop** [comment]
Published in f+/cli v0.7.32 using generator v0.6.31
